admin/acceptance_tests/edit_url_test.py

```diff
@@ -104,6 +104,7 @@ class TestUrlEdit(AbstractViewsTests):
 
     def _get(self, test_app, tablename, pk):
         path = '/{}/{}'.format(tablename, pk)
+        print(path)
```

Comment:
A logger should be used, and setting one should be in my opinion of great value.